packages/ses-ava/src/ses-ava-test.js

```diff
@@ -23,21 +14,50 @@ const isPromise = maybePromise =>
   Promise.resolve(maybePromise) === maybePromise;
 
 /**
- * @type {LogCallError}
+ * @typedef {(...args: unknown[]) => void} Logger
  */
-const logErrorFirst = (func, args, name, logger = defaultLogger) => {
+
+const defaultLogError = (...args) => console.error(...args);
+
+/**
+ * Calls `func(...args)` passing back approximately its outcome, but first
+ * logging any erroneous outcome to the `logger`, which defaults to
```

Comment:
since this only logs to one place, I think the earlier `logger` name is more appropriate. That it's logging an error is implied by it being the `logger` param of `logErrorFirst`.

Regardless,
1. the name here in the docs should match the param
2. the default is `console.error` not `console.log`

Reply:
Ha, didn't realize that was removed and then subsequently restored! Fixed.